scripts/release.sh

```diff
@@ -437,6 +445,9 @@ function parse_flags() {
   [[ -n "${RELEASE_VERSION}" ]] && TAG="v${RELEASE_VERSION}"
   [[ -n "${RELEASE_VERSION}" && -n "${RELEASE_BRANCH}" ]] && (( PUBLISH_RELEASE )) && PUBLISH_TO_GITHUB=1
 
+  # If not already set then go ahead and default to gcr
+  export KO_DOCKER_REPO="${KO_DOCKER_REPO:-gcr.io/knative-nightly}"
+
```

Comment:
Have to protect this with if NOT (( USE_KO_DOCKER_REPO )) then export KO_DOCKER_REPO="gcr.io/knative-nightly" done

Reply:
I think this needs to be before the check for PUBLISH_RELEASE, so I moved it but check me on that...